k8s/helmfile/env/production/prometheus-elasticsearch-exporter-1.values.yaml.gotmpl

```diff
@@ -0,0 +1,23 @@
+es:
+  uri: http://elasticsearch-1.default.svc.cluster.local:9200
```

Comment:
So we gave up on the idea of using a headless service because we just don't know which one to monitor, correct?

Reply:
I wonder if we can find out where this service is pointing at?